recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2019.07.15" %}
+{% set version = "2019.07.19" %}
```

Comment:
We need to bump this b/c I merged #254 first.

Reply:
I've bumped the build number, or would you prefer I change the date to tomorrow?